enhancements/microshift/audit-log-configuration-options.md

```diff
@@ -0,0 +1,255 @@
+
+**Audit Log Rotation**
+
+The kube-apiserver provides three CLI flags to dictate log rotation and retention policies which may be thinly exposed to the user via the MicroShift config API. These options will be set dynamically, depending on user configuration in the MicroShift config. Logic already exists to set these flags with hardcoded values, which will need to be updated to allow for dynamic arguments.  These options are:
+
+* `--audit-log-maxage` defines the maximum number of days to retain old audit log files.  MicroShift defaults to 10 days.
+* `--audit-log-maxbackup` defines the maximum number of audit log files to retain. MicroShift defaults to 10 files.
+* `--audit-log-maxsize` defines the maximum size in megabytes the live log file before it is rotated.  Defaults to 200Mb.
+
+Thus, the default maximum storage consumption of audit logs will be 2000Mb, provided all files are younger than 10 days.
+
+**Policy Profiles**
+
+OpenShift's policy profiles are defined as part of the `openshift-cluster-config-operator` API and are not recognized by the kube-apiserver. A profile can be considered a higher level implementation of the `policy.audit.k8s.io/v1` API, where each profile maps to a predefined `policy`. A `Policy.audit.k8s.io/v1` object consists of a set of rules, with each rule specifying the API resources, verbs, and level. Policy objects must be defined by the user and thus require a familiarity with Kubernetes APIs and auditing events, and the user's target security posture.  There are no safeguards that prevent users from inadvertently exposing sensitive information in logs. Below is a trivial example of a `policy` object.
+
+```yaml
```

Comment:
We're not going to expose this policy API directly, right? If not, I think we can eliminate the example here to avoid confusion later when we come to test and document the change.

Reply:
removed the example